src/codegen/index.ts

```diff
@@ -11,6 +11,7 @@ export type Opts = {
   include?: string[];
   exclude?: string[];
   optimistic?: boolean;
+  unionUndefined?: boolean;
```

Comment:
Would you mind adding a doc block quickly above this for the intended purpose? Maybe something like:

> Useful for when `exactOptionalPropertyTypes: true` in `tsconfig`. Produces `prop?: string | undefined` instead of `prop?: string`

Reply:
Thx! Done 👍